src/LavaNode.cs

```diff
@@ -555,10 +552,13 @@ await _webSocketClient.DisposeAsync()
                 }
             }
         }
-        catch (Exception exception) {
-            _logger.LogError(exception is JsonReaderException
-                ? "Please increase buffer size in configuration."
-                : $"{exception.Message} {exception}");
+        catch (JsonException jsonException)
```

Comment:
Use pattern match to handle different error message.

```cs
catch (Exception exception) {
 _logger.LogError(exception is JsonException ? "Increase buffer size" : "normal message");
}
```

but there should be no need for this, going to replace buffer with memory stream